aee.js

```diff
@@ -72,3 +72,13 @@ Array.prototype.count = function (spec) {
   }
   return result
 }
+
+Array.prototype.pluck = function (property) {
+  if (property === undefined) throw new ReferenceError('property is undefined')
+  const len = this.length
+  let result = []
+  for (let i = 0; i < len; i += 1) {
+    result.push(this[i] && this[i].hasOwnProperty(property) ? this[i][property] : null)
```

Comment:
Runned the tests, and it starts throwing `undefined` in objects where the property is not found. Should this be the correct behaviour? Right now it sends null both when an element is null, or if the object lacks the property sent to the function.